perses/tests/utils.py

```diff
@@ -58,6 +58,13 @@ def extractPositionsFromOEMOL(molecule):
         positions[index,:] = unit.Quantity(coords[index], unit.angstroms)
     return positions
 
+def giveOpenmmPositionsToOEMOL(positions, molecule):
+    assert molecule.NumAtoms() == len(positions)
+    coords = molecule.GetCoords()
+    for key in coords.keys():
+        coords[key] = (positions._value[key][0],positions._value[key][1],positions._value[key][2])
```

Comment:
This is very bad!  It just takes the value without converting!

Can you use `positions[key][0] / unit.angstroms`?